### PR TITLE
Fix minor issues in the slides of mapping tutorial

### PR DESCRIPTION
**Pull request by anuprulez**

### CODE REVIEW

File: topics/sequence-analysis/tutorials/mapping/slides.html [hunk 1]
```diff
@@ -964,7 +964,7 @@
 
 *“... there is no tool that outperforms all of the others in
 all the tests. Therefore, the end user should clearly
-specify his needs in order to choose the tool that
+specify [her/]his needs in order to choose the tool that
```
Comment (nsoranzo) [inline topics/sequence-analysis/tutorials/mapping/slides.html]:
```suggestion
specify [their] needs in order to choose the tool that
```

Comment (anuprulez) [reply topics/sequence-analysis/tutorials/mapping/slides.html]: @nsoranzo if we use `their` then we should also modify at the start of the sentence which reads as `the end user ...` and change to `the end users ...`

Comment (nsoranzo) [reply topics/sequence-analysis/tutorials/mapping/slides.html]: @anuprulez No, it's a [singular their](https://en.wikipedia.org/wiki/Singular_they).

Comment (anuprulez) [reply topics/sequence-analysis/tutorials/mapping/slides.html]: @nsoranzo thanks for your suggestion :)

**Comment by shiltemann** [comment]
nicely spotted @anuprulez thanks!

**Pull request merged**